tools/accuracy_checker/configs/openpose-cmu-tf.yml

```diff
@@ -0,0 +1,54 @@
+models:
+  - name: openpose-cmu-tf
+
+    launchers:
+      - framework: tf
+        model: public/openpose-cmu-tf/opt_graph.pb
+        adapter:
+          type: human_pose_estimation
+          part_affinity_fields_out: Mconv7_stage6_L1/Conv2D
+          keypoints_heatmap_out: Mconv7_stage6_L2/Conv2D
+
+    datasets:
+      - name: ms_coco_keypoints
+        preprocessing:
+          - type: resize
+            size: 368
+            interpolation: CUBIC
+            aspect_ratio_scale: width
+          - type: padding
+            stride: 8
+
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/openpose-cmu-tf/FP32/opt_graph.bin
```

Comment:
I am not sure about converted model name correctness... As I see it build based on model directory name.
so possibly it is
`public/openpose-cmu-tf/FP32/openpose-cmu-tf.xml` ?

Reply:
yes, it is true, look on model downloader precommit logs:
Conversion command: venv/conversion/bin/python -- /opt/intel/openvino/deployment_tools/model_optimizer/mo.py --framework=tf --data_type=FP32 --output_dir=download/public/openpose-cmu-tf/FP32 --model_name=openpose-cmu-tf '--input_shape=[1,368,432,3]' --input=image --output=Mconv7_stage6_L2/Conv2D,Mconv7_stage6_L1/Conv2D --input_model=download/public/openpose-cmu-tf/graph_opt.pb